transport-native-kqueue/pom.xml

```
@@ -325,7 +325,7 @@
                       <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
                     </manifest>
                     <manifestEntries>
-                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_kqueue_aarch_64.jnilib; osname=MacOSX; processor=aarch64</Bundle-NativeCode>
+                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_kqueue_x86_64.jnilib; osname=MacOSX; processor=x86_64</Bundle-NativeCode>
                       <Fragment-Host>io.netty.transport-classes-kqueue</Fragment-Host>
```

I think we need `netty5` in the name for OSGi metadata. Wdyt?
The same is applicable also for `resolver-dns-native-macos`.

```suggestion
                      <Bundle-NativeCode>META-INF/native/libnetty5_transport_native_kqueue_x86_64.jnilib; osname=MacOSX; processor=x86_64</Bundle-NativeCode>
                      <Fragment-Host>io.netty5.transport-classes-kqueue</Fragment-Host>
```